radix-engine-common/src/macros.rs

```diff
@@ -246,6 +246,13 @@ macro_rules! manifest_args {
     }};
 }
 
+#[macro_export]
+macro_rules! manifest_args {
+    ($($args: tt)*) => {{
+        $crate::data::manifest::manifest_decode(&manifest_vec!($($args)*)).unwrap()
+    }};
```

Comment:
Can we merge `manifest_vec` into `manifest_args`?

Reply:
Yeah, that was my initial plan, but it turned there are some places here and there (eg. [transaction/src/manifest/generator.rs line 1582](https://github.com/radixdlt/radixdlt-scrypto/pull/891/files#diff-d0f0e4e06a13314ed475969bc3c09ef0bfc28668ae30a1d4d89fe78c126f3887R1582)) where Vec<u8> is required.

Reply:
Let's remove the commented code here.